lib/src/main/java/com/orange/ods/compose/component/list/OdsListItem.kt

```diff
@@ -311,6 +341,100 @@ fun Modifier.iconType(iconType: OdsListItemIconType): Modifier {
     })
 }
 
+/**
+ * A modifier that allows to configure the icon type in an [OdsListItem].
+ */
+private interface OdsListItemIconTypeModifier : Modifier.Element {
+    val iconType: OdsListItemIconType
+}
+
+//endregion
+
+//region OdsListItem Trailing
+
+private fun Modifier.trailingModifier(trailing: OdsListItemTrailing): Modifier = when (trailing) {
```

Comment:
I think this could be renamed to `trailing` because other `Modifier` methods are not suffixed with `modifier`.